documentation/project-docs/breaking-change-guidelines.md

```diff
@@ -84,5 +85,5 @@ Specific example: NuGet warnings for vulnerable transitive dependencies were int
 
 * Create an issue in the appropriate GitHub repository to track the change, if one does not already exist.
 * Add the breaking-change label to the issue. This label should be available in all .NET repositories that ship as part of the .NET SDK. If the label is not available, please file an issue in [dotnet/sdk](https://github.com/dotnet/sdk).
-* The issue will trigger a message with instructions to add documentation. In addition, you are invited to work with the SDK team to publish a blog post for the change. 
 * Consider creating and pinning an issue in the appropriate GitHub repository where the community can provide feedback.
+* Once a Pull Request is submitted for the issue, add the breaking-change label to the Pull Request as well. This will trigger a message with instructions. In addition, you are invited to work with the SDK team to publish a blog post for the change. 
```

Comment:
Should there be a link to the docs repo where the breaking change would be filed or leave that to the instructions?